src/extensions/ITemplatePlugin.ts

```diff
@@ -0,0 +1,7 @@
+import { PluginUtilities, XmlNode, PluginContent } from "src";
```

Comment:
Please remove this file.

This is too specific in my opinion. I wouldn't want to force every extension author to use plugins. Maybe he has a very simple implementation that only edit one xml file (a good example would be an extension that edits some document metadata in "docProps/core.xml"), that would be an overkill to use plugins in this case.

Reply:
I kept it for consistency happy to move to the specific extensions.